.github/workflows/ci.yml

```
@@ -6,19 +6,19 @@ on:
       - "**"
 jobs:
   build:
-     name: Maven Build
-     runs-on: ubuntu-latest
-     steps:
-       - uses: actions/checkout@v3
+    name: Maven Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)